x-pack/plugins/fleet/public/applications/integrations/sections/epm/constants.tsx

```diff
@@ -66,10 +66,10 @@ export const AssetTitleMap: Record<
     }
   ),
   'ml-module': i18n.translate('xpack.fleet.epm.assetTitles.mlModules', {
-    defaultMessage: 'ML modules',
+    defaultMessage: 'Anomaly Detection supplied configurations',
```

Comment:
I think we can drop 'supplied' from this text to the more concise `Anomaly detection configurations`.

Reply:
updated in 3c72ce197202467effe812c84029eb2b3ca89b53